kernelci/build.py

```diff
@@ -802,6 +802,28 @@ def _kernel_config_enabled(self, config_name):
         cmd = 'grep -cq CONFIG_{}=y {}'.format(config_name, dot_config)
         return shell_cmd(cmd, True)
 
+    def _kernel_config_getkey(self, opt_name):
+        dot_config = os.path.join(self._output_path, '.config')
+        with open(dot_config, 'r') as fp:
+            lines = fp.readlines()
+            for line in lines:
+                pcfg = line.strip().split('=')
```

Comment:
You might also do `if line.startswith(opt_name)` before doing the split.  That way you don't have to split all the lines until you've found the matching one.  But that's just an implementation / performance tweak.

Reply:
Done